app/assets/stylesheets/application_serif.scss

```diff
@@ -0,0 +1,11 @@
+/*
```

Comment:
It might be good to create a few different themes and share with the spotlight committee for feedback on staging (or via a demo since otherwise they'll have to create a stg account) before we deploy to prod? What do you think?

Reply:
Sure, we can wait to merge this until after the prod deploy this week, then.